src/displayapp/screens/Timer.cpp

```diff
@@ -10,62 +10,32 @@ static void btnEventHandler(lv_obj_t* obj, lv_event_t event) {
   screen->OnButtonEvent(obj, event);
 }
 
-void Timer::CreateButtons() {
-  btnMinutesUp = lv_btn_create(lv_scr_act(), nullptr);
-  btnMinutesUp->user_data = this;
-  lv_obj_set_event_cb(btnMinutesUp, btnEventHandler);
-  lv_obj_set_size(btnMinutesUp, 60, 40);
-  lv_obj_align(btnMinutesUp, lv_scr_act(), LV_ALIGN_IN_LEFT_MID, 20, -85);
-  txtMUp = lv_label_create(btnMinutesUp, nullptr);
-  lv_label_set_text_static(txtMUp, "+");
+Timer::Timer(DisplayApp* app, Controllers::TimerController& timerController) : Screen(app), timerController {timerController} {
```

Comment:
nitpick: this line is SO LONG! Can we update the clang formatter to make constructor elements be on their own lines? Makes diffs easier as well in my opinion

Reply:
This is controlled by ColumnLimit, which is 140. I might also like to decrease it, however that's not in the scope of this PR.